framework/test/integration/node/processor/pos.spec.ts

```diff
@@ -224,7 +224,12 @@ describe('PoS and reward', () => {
 			newBlock = await processEnv.createBlock([claimTx]);
 			await processEnv.process(newBlock);
 
-			expect(claimableRewards.rewards[0].reward).toEqual(lockedReward.reward);
+			expect(lockedReward.reward).toBe('500000000');
+			expect(claimableRewards.rewards[0].reward).toBe('499999999');
```

Comment:
I don't see why there is a difference here. Seems to be that they should be exactly the same..

Reply:
locked reward is using `ceil` so it should result in 5, while for claimableRewards it is using `floor`, so it is `499999999`. 
this happens because of 
```
const rewardFractionQ = q96(BigInt(1)).sub(commissionQ.div(q96(BigInt(10000))));
const sharingCoefficientIncrease = rewardQ.muldiv(rewardFractionQ, totalStakesQ);
```
division by total stake shile computing sharing coefficient